File: leetcode/0-250/120-generate.java
    public static List<List<Integer>> generate(int numRows) {
        List<List<Integer>> pt = new ArrayList<>();
        for (int i = 0; i < numRows; i++) {
            List<Integer> lev = new ArrayList<>();
            lev.add(1);
            for (int j = 1; j < i; j++) {
                lev.add(pt.get(i - 1).get(j) + pt.get(i - 1).get(j - 1));
            }
            if (i != 0) lev.add(1);
            pt.add(lev);
        }
        return pt;
    }

File: leetcode/0-250/120-generate.py
def generate(self, numRows):
    pt = []
    for i in range(numRows):
        lev = []
        lev.append(1)
        for j in range(1, i):
            lev.append(pt[i - 1][j] + pt[i - 1][j - 1])
        if i != 0: lev.append(1)
        pt.append(lev)
    return pt